templates/pkgs/base.json

```diff
@@ -49,5 +57,10 @@
   },
   "commitlint": {
     "extends": ["@commitlint/config-conventional"]
+  },
+  "prettier": {
+    "printWidth": 120,
+    "singleQuote": true,
+    "trailingComma": "es5"
```

Comment:
```suggestion
    "trailingComma": "all"
```

Reply:
Any particular reason?

Reply:
Trailing commas in functions :)

Reply:
Hmm like this or something?

```js
const myFunc = (foo, bar,) => {
}
```

Reply:
Yeah, multiline only - results in smaller noise in diff.
It also works for function calls, i.e.
```
someReallyLongFunctionName(
  someReallyReallyReallyLongVariableName,
  someReallyReallyReallyReallyReallyLongVariableName,
);
```

It was added in ES2017 if I am not mistaken, hence it's not enabled by default as it causes a syntax error in ancient engines/parsers. We aren't really affected since we transpile our code and don't really aim to support ES5 environments anyway.